src/cmd-fetch

```diff
@@ -55,9 +55,18 @@ lock_arg=
 if [ -n "${UPDATE_LOCKFILE}" ]; then
     # Put this under tmprepo so it gets automatically chown'ed if needed
     lock_arg="--ex-write-lockfile-to=${tmprepo}/tmp/manifest-lock.json"
+    if [ -f "${manifest_lock_overrides}" ]; then
+        echo "NB: ignoring overrides ${manifest_lock_overrides}"
+        sleep 1
```

Comment:
why the sleep?

Reply:
Just to give time to read the message. It might not be obvious how `cosa fetch --update-lockfile` behaves in the presence of overrides.